_episodes/13-looking-up-data.md

```diff
@@ -144,7 +144,7 @@ The functionality in OpenRefine can be enhanced by ‘extensions’ which can be
 
 A list of Extensions (not necessarily complete) is given on the OpenRefine downloads page at [http://openrefine.org/download.html](http://openrefine.org/download.html)
 
-One of these extensions tries to work around the limitation of Reconciliation services described above, by making it possible to use a reconciliation service against ‘linked data’ sources which have SPARQL endpoints. For more information on this see the ‘RDF Extension’ at [http://refine.deri.ie](http://refine.deri.ie). An example of how this works is given in more detail at [http://refine.deri.ie/showcases](http://refine.deri.ie/showcases).
+One of these extensions tries to work around the limitation of Reconciliation services described above, by making it possible to use a reconciliation service against ‘linked data’ sources which have SPARQL endpoints. For more information on this see the ‘RDF Extension’ at [http://refine.deri.ie](https://web.archive.org/web/20180112163745/http://refine.deri.ie/) (Available at Internet Archive Wayback Machine). An example of how this works is given in more detail at [http://refine.deri.ie/showcases](https://web.archive.org/web/20180113135530/http://refine.deri.ie/showcases) (Available at Internet Archive Wayback Machine).
```

Comment:
I'm not sure we need this paragraph anymore. The DERI information is out of date and the previous para already says what the RDF extension does

Reply:
Thanks @ostephens. I don't think we need it either. I removed it.